lib/dynamoid/criteria/chain.rb

```diff
@@ -128,6 +128,10 @@ def each(&block)
         records.each(&block)
       end
 
+      def find_by_pages(&block)
```

Comment:
What is the final verdict on the  page for this, if you would prefer pages I can move that method up

Reply:
The name `find_by_pages` looks good.